conf/st2.dev.conf

```diff
@@ -64,7 +64,7 @@ port = 6001
 url = /webhooks/generic/
 
 [action_sensor]
-triggers_base_url = http://localhost:9101/triggertypes/
+triggers_base_url = http://localhost:9101/v1/triggertypes/
```

Comment:
Add the v1. No worries. the trailing slash should not be required since we use urljoin but why bother with weird regressions.